### PR TITLE
fix(activity): normalize decimal points display

### PR DESCRIPTION
**Pull request by chibie**
##### Description

normalize decimal points for token value in grant contribution activity.

previously, `0.001` will be displayed as `0.00`

now, `0.001` will show `0.001`, `0.002000` will show `0.002` without the trailing zeroes.

### CODE REVIEW

**Review by thelostone-mc — APPROVED** [review]

**Review by aamustapha — APPROVED** [review]

File: app/retail/templates/shared/activity-vue.html [hunk 1]
```diff
@@ -332,15 +332,15 @@
...
+                    <span class="value_in_token_bounty">{{ row.bounty.value_true.normalize }} {{row.bounty.token_name}}</span>
                   {% elif row.value_in_token_disp %}
-                    <span class="value_in_token_disp">{{ row.value_in_token_disp | floatformat:2 }} {{row.token_name}}</span>
+                    <span class="value_in_token_disp">{{ row.value_in_token_disp.normalize }} {{row.token_name}}</span>
                   {% elif row.metadata.value_in_eth %}
-                    <span class="value_in_eth">{{ row.metadata.value_in_eth | floatformat:2 }} ETH</span>
+                    <span class="value_in_eth">{{ row.metadata.value_in_eth.normalize }} ETH</span>
                   {% elif row.metadata.value_in_token and row.metadata.token_name %}
-                    <span class="value_in_token">{{ row.metadata.value_in_token | floatformat:2}} {{row.metadata.token_name}}</span>
+                    <span class="value_in_token">{{ row.metadata.value_in_token.normalize }} {{row.metadata.token_name}}</span>
```
Comment (octavioamu) [inline app/retail/templates/shared/activity-vue.html:343]: Is this file used?

**Review by octavioamu — CHANGES_REQUESTED** [review]

File: app/retail/templates/shared/activity-vue.html (hunk 1 above)
Comment (thelostone-mc) [reply app/retail/templates/shared/activity-vue.html:343]: doesn't look like it ! nice catch @octavioamu 
can you double check and remove this file @chibie  ?

Comment (chibie) [reply app/retail/templates/shared/activity-vue.html:343]: @thelostone-mc @octavioamu should i delete it or revert the changes to it?

Comment (thelostone-mc) [reply app/retail/templates/shared/activity-vue.html:343]: delete

**Pull request merged**